app/controllers/images_controller.rb

```diff
@@ -35,12 +36,19 @@ def create
   # PATCH/PUT /images/1
   def update
     if @image.update(image_params)
-      redirect_to artwork_images_url(@artwork), notice: "Image was successfully updated.", status: :see_other
+      redirect_to @artwork
```

Comment:
This change is for image caption editing. Since the only form using the update is the one I just added I change the redirect to the artwork page to have the edit happen without refreshing the whole page.

Reply:
Why do we need to skip the auth token for the JS call?